src/allotropy/parser_factory.py

```diff
@@ -29,10 +32,11 @@
 
 class Vendor(Enum):
     AGILENT_GEN5 = "AGILENT_GEN5"
-    APPBIO_QUANTSTUDIO = "APPBIO_QUANTSTUDIO"
     APPBIO_ABSOLUTE_Q = "APPBIO_ABSOLUTE_Q"
+    APPBIO_QUANTSTUDIO = "APPBIO_QUANTSTUDIO"
```

Comment:
Thanks for doing this, but unrelated cleanup should really be in a separate PR. No need to remove -- I'm doing here: #155

Reply:
tbh, I didn't realize I did this.